plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -182,15 +185,12 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
 
     private fun processProperty(symbol: FirPropertySymbol) {
         val name = symbol.callableId.embedMemberPropertyName()
-        val backingField = symbol.hasBackingField.ifTrue {
-            FieldEmbedding(symbol.callableId.embedMemberPropertyName(), embedType(symbol.resolvedReturnType))
-        }
-
+        val backingField = symbol.specialEmbedding() ?: symbol.hasBackingField
```

Comment:
The precedence here is a bit confusing.  Do I understand correctly that the following lines are only executed if it *doesn't* have a special embedding?

Reply:
yes, is like 
```kotlin
if(symbol.specialEmbedding() != null) {
  symbol.specialEmbedding()
} else {
symbol.hasBackingField
    .ifTrue { FieldEmbedding(symbol.callableId.embedMemberPropertyName(), embedType(symbol.resolvedReturnType)) }
    .also { it?.let { fields.add(it) } }
}
```
I'll try to write in clearer way

Reply:
As an alternative solution: perhaps instead of using a special embedding, you could force the system to believe this property *does* have a backing field?  Though admittedly this may be a bit painful.  Either way is fine.